lib/irc/ClientPool.js

```diff
@@ -204,6 +204,20 @@ ClientPool.prototype._getNumberOfConnections = function(server) {
     return numConnectedNicks;
 };
 
+ClientPool.prototype.countTotalConnections = function() {
+    var count = 0;
+
+    Object.keys(this._virtualClients).forEach((domain) => {
```

Comment:
In 99% of cases, there will be precisely 1 server, so linear scans are not a problem. I agree the back and forth from a `server` is excessive, so am happy for you to make `_getNumberOfConnections` take a server domain instead of a server.